source/_integrations/deconz.markdown

```diff
@@ -40,6 +40,8 @@ There is currently support for the following device types within Home Assistant:
 An official add-on for deCONZ is available in the Home Assistant add-on store.
 Otherwise, use [community container](https://hub.docker.com/r/marthoc/deconz/) by Marthoc for your deCONZ needs.
 
+Connecting deCONZ to Home Assistant requires both the add-on and the deCONZ integration.
```

Comment:
This is actually incorrect, the deCONZ application this integration connects to can be run in different ways. The add-on is one of them.

Reply:
Ok. Do you have any suggestions on how to make this info more beginner friendly? What I did was literally just plugged in my Conbee II and the add-on appeared. And after that, the Integration was found as well. So I assumed it's the default behavior for hass.io.

Reply:
I think it is beginner-friendly? The first paragraph explains this integration connects to deCONZ software created by dresden:

![image](https://user-images.githubusercontent.com/195327/100518964-3d1b0080-3195-11eb-9a90-46b50c563c8e.png)

This paragraph suggests that there are options for that available when running Docker or a supervisor enabled system (using add-ons).

So, I'm not sure what is missing / needs to be added here?